verifyFiles.py

```diff
@@ -537,17 +553,12 @@ def main():
 	#and the file checksumFile,checkCorruption and the path to the sqlite file.
         #diff,metric_values,dictionary_executables,dictionary_processes=n_differences_across_subjects(conditions_dict,root_dir,metrics,checksums_from_file_dict,args.checksumFile,args.checkCorruption,args.sqLiteFile)i
 	diff,bDiff,metric_values,dictionary_executables,dictionary_processes,metric_values_subject_wise=n_differences_across_subjects(conditions_dict,root_dir,metrics,checksums_from_file_dict,args.checksumFile,args.checkCorruption,args.sqLiteFile,args.trackProcesses)
-       	if args.fileDiff is not None:
-            log_info("Writing difference matrix to file "+args.fileDiff)
-            diff_file = open(args.fileDiff,'w')
+       	if args.difference is not None:
+            log_info("Writes the difference matrices and indexs into files"+args.difference)
```

Comment:
args.difference is the base name of the file. It would be better to edit the log info to a  more meaningful one.